pkg/util/translate/translate.go

```diff
@@ -325,32 +351,30 @@ func ConvertKindToResource(config *rest.Config, groupVersionKind schema.GroupVer
 	return schema.GroupVersionResource{}, kerrors.NewNotFound(schema.GroupResource{Group: groupVersionKind.Group}, groupVersionKind.Kind)
 }
 
-// KindExistsAndIsClusterScoped checks if given CRDs exist in the given group.
+var ErrKindNotFound = errors.New("kind not found")
+
+// KindExists checks if given CRDs exist in the given group.
 // Returns foundKinds, notFoundKinds, error
```

Comment:
nit: comment doesn't match return struct

Reply:
Yeah please change that then we can merge it

Reply:
Good catch. Fixed. Thanks.